python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -604,6 +604,3 @@ def func(X):
 grid.setGraph(1, 1, graph)
 
 _ = View(grid)
-
-
-View.ShowAll()
```

Comment:
this should not be removed (restore it in a new cell with %%)

Reply:
sorry... It is done!